catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/WfsResponseExceptionMapper.java

```diff
@@ -46,7 +46,7 @@ public WfsException fromResponse(Response response) {
           is.reset();
           msg = IOUtils.toString(is);
         } catch (IOException e) {
-          wfsEx = new WfsException("Error reading Response" + (msg != null ? ": " + msg : ""), e);
```

Comment:
What was the finding here?  It might be nice to preserve the message in this exception

Reply:
`msg` will always be null, so it will never print out the additional info